doc/installationguide.md

```diff
@@ -180,6 +180,7 @@ The following table shows the accepted environment variables, as well as the con
 | IOTA_AUTH_PORT            | authentication.port                 |
 | IOTA_AUTH_USER            | authentication.user                 |
 | IOTA_AUTH_PASSWORD        | authentication.password             |
+| IOTA_AUTH_PERMANENT_TOKEN | authentication.permanentToken       |
```

Comment:
The `IOTA_AUTH_PERMANENT_TOKEN`, `IOTA_AUTH_PASSWORD` and `IOTA_AUTH_USER` are all sensitive items of information, and therefore should also be available as *Docker Secrets*

It shouldn't affect the PR here, but the Docker containers for the IoT Agents will need to handle loading passwords from an encrypted file.

An example `entrypoint.sh` which handles Docker secrets can be found within PostGres 

https://github.com/docker-library/postgres/blob/040949af1595f49f2242f6d1f9c42fb042b3eaed/11/docker-entrypoint.sh#L5-L25